### ISSUE TITLE
catch22 situation with tf.nn.sampled_softmax_loss and tf.nn.softmax

### ISSUE THREAD

**Issue by vince62s** [tensorflow/tensorflow#5890]
Hi,
referring to this: 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/api_docs/python/functions_and_classes/shard1/tf.nn.sampled_softmax_loss.md
and this:
https://github.com/tensorflow/tensorflow/issues/4138

to make samlped_softmax efficient we need to invert w and w_t to take transpose out of the loss function.
but then it becomes an issue at test / inference time when you need to use the full softmax with the tranpose inside.

ie: either training is slow or testing is slow.

what's the solution ?

**Comment by prb12** [comment]
From https://www.tensorflow.org/versions/r0.11/resources/index.html
> For help and support, technical or algorithmic questions, please submit your questions to Stack Overflow: https://stackoverflow.com/questions/tagged/tensorflow. You may also find answers in our FAQ, our glossary, or in the shapes, sizes and types guide. Please do not use the mailing list or issue tracker for support.
